tests/test_visitors/test_ast/test_complexity/test_counts/test_function_output_length.py

```diff
@@ -1,18 +1,21 @@
 import pytest
 
-from wemake_python_styleguide.constants import MAX_LEN_YIELD_TUPLE
+from wemake_python_styleguide.constants import MAX_LEN_TUPLE_OUTPUT
 from wemake_python_styleguide.violations.complexity import (
-    TooLongYieldTupleViolation,
+    TooLongFunctionOutputTupleViolation,
```

Comment:
Let's name it `TooLongOutputTupleViolation`

Reply:
Done. Also renamed test file accordingly.